app/views/admin/tags/show.html.haml

```diff
@@ -11,7 +11,7 @@
 
 .dashboard
   .dashboard__item
-    = react_admin_component :counter, measure: 'tag_accounts', start_at: @time_period.first, end_at: @time_period.last, params: { id: @tag.id }, label: t('admin.trends.tags.dashboard.tag_accounts_measure')
+    = react_admin_component :counter, measure: 'tag_accounts', start_at: @time_period.first, end_at: @time_period.last, params: { id: @tag.id }, label: t('admin.trends.tags.dashboard.tag_accounts_measure'), href: tag_url(@tag), target: '_blank', rel: 'noopener noreferrer'
```

Comment:
`target: '_blank'` is specified, but for some reason it opens in the same tab. 🤔

Reply:
The `href` is passed to `app/javascript/mastodon/components/admin/Counter.js` through `data` attribute, there is no non-JS fallback, and there is no handling for a `target` or a `rel` attribute.

Reply:
Thank you!
I need to remove the `target: '_blank', rel: 'noopener noreferrer'` or change it to work properly, right? Which one should I choose?
I think it's better to remove it considering the impact on others.

Reply:
I wouldn't really bother with `noopener noreferrer` since everything is on the same instance/server. Adding a `target: '_blank'` may make sense, I'd just add `target` support to `app/javascript/mastodon/components/admin/Counter.js` in that case.

Reply:
Thank you! I will try it that way.